src/pip/_internal/commands/cache.py

```diff
@@ -93,25 +93,31 @@ def get_cache_info(self, options: Values, args: List[Any]) -> None:
         num_http_files = len(self._find_http_files(options))
         num_packages = len(self._find_wheels(options, "*"))
 
-        http_cache_location = self._cache_dir(options, "http")
+        http_cache_location = self._cache_dir(options, "http-v2")
+        old_http_cache_location = self._cache_dir(options, "http")
         wheels_cache_location = self._cache_dir(options, "wheels")
         http_cache_size = filesystem.format_directory_size(http_cache_location)
+        old_http_cache_size = filesystem.format_directory_size(old_http_cache_location)
         wheels_cache_size = filesystem.format_directory_size(wheels_cache_location)
 
         message = (
             textwrap.dedent(
                 """
-                    Package index page cache location: {http_cache_location}
-                    Package index page cache size: {http_cache_size}
-                    Number of HTTP files: {num_http_files}
+                    Package index page cache location (new): {http_cache_location}
+                    Package index page cache location (old): {old_http_cache_location}
+                    Package index page cache size (new): {http_cache_size}
+                    Package index page cache size (old): {old_http_cache_size}
+                    Number of HTTP files (old+new cache): {num_http_files}
```

Comment:
We probably should mention this old-new difference in the documentation. This would be a bit confusing to users. (Or maybe we can just merge them? What is the use case to show these as separate entries?)

Reply:
Use case is if there is way to manage the two caches separately. Which at the moment ... there isn't? So maybe worth merging yeah if we stick to that.

Reply:
I ended up only merging the size. having different locations mentioned is actually explained in the docs, and it is useful if people say "oh I am using new pip so can delete old cache manually". I did however improve the explanation so its clearer what new vs old means.